<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGraphWindow</class>
 <widget class="QDialog" name="CreateGraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ввод графа</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/input.png</normaloff>:/icons/input.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0,0,0);
background-color: rgb(232, 232, 232);
</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>401</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font: 12pt &quot;Cascadia Code&quot;;
background-color: rgb(232, 232, 232);
border-color: rgb(0, 0, 0);
border-radius: 10px;
border: 1px solid grey;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string>color: rgb(0,0,0);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="formatLabel">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0,0,0);
border-color: rgb(200, 200, 200);
border-radius: 15px;
border: 1px solid;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Формат ввода&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="readMatrixButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(0,0,0);
border-color: rgb(0, 0, 0);
border-radius: 5px;
border: 1px solid grey;</string>
      </property>
      <property name="text">
       <string>Матрица смежности</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">formatButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="readEdgesButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(0,0,0);
border-color: rgb(0, 0, 0);
border-radius: 5px;
border: 1px solid grey;</string>
      </property>
      <property name="text">
       <string>Список ребер</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">formatButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="readAdjacencesButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(0,0,0);
border-color: rgb(0, 0, 0);
border-radius: 5px;
border: 1px solid grey;</string>
      </property>
      <property name="text">
       <string>Список связности</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">formatButtonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="readIncidenceMatrixButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(0,0,0);
border-color: rgb(0, 0, 0);
border-radius: 5px;
border: 1px solid grey;</string>
      </property>
      <property name="text">
       <string>Матрица инцедентности</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">formatButtonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="acceptButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
color: rgb(0,0,0);
border-color: rgb(0, 0, 0);
border-radius: 10px;
border: 1px solid grey;
}
QPushButton:hover
{
	background-color:rgb(150, 150, 150);
}
QPushButton:pressed
{
	background-color:rgb(100, 100, 100);
}</string>
      </property>
      <property name="text">
       <string>ОК</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
color: rgb(0,0,0);
border-color: rgb(0, 0, 0);
border-radius: 10px;
border: 1px solid grey;
}
QPushButton:hover
{
	background-color:rgb(150, 150, 150);
}
QPushButton:pressed
{
	background-color:rgb(100, 100, 100);
}</string>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="formatButtonGroup"/>
 </buttongroups>
</ui>
